lib/src/container/encapsulate.rs

```diff
@@ -68,7 +70,7 @@ fn commit_meta_to_labels<'a>(
     #[allow(clippy::explicit_auto_deref)]
     if let Some(v) = meta.lookup::<bool>(*ostree::METADATA_KEY_BOOTABLE)? {
         labels.insert(ostree::METADATA_KEY_BOOTABLE.to_string(), v.to_string());
-        labels.insert("containers.bootc".into(), "1".into());
+        labels.insert(BOOTC_LABEL.into(), "1".into());
```

Comment:
What is `v` in this context? Is it just a version number?

Reply:
short for "value"

Reply:
But it's always a boolean here

Reply:
Ah, gotcha. Just couldn't figure out what it was short for.